cmd/kops/create_cluster.go

```diff
@@ -844,6 +848,17 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		Legacy: false,
 	}
 
+	// Check if encryption-at-rest for etcd is enabled.
+	if c.EncryptEtcdData {
+		cluster.Spec.KubeEtcdEncryptionConfig = &api.KubeEtcdEncryptionConfig{
```

Comment:
If we do keep the cli options, in regard to the spec variable, the command line flag experimental-encryption-provider-config doesn't mention etcd (albeit, yes it's not only backend for k8s at present); still i wonder if we should keep the naming abstract ```s/KubeEtcdEncryptionConfig/KubeEncryptionProviderConfig/```

Reply:
Yes, I wondered about that too, I'll remove the references to etcd.